bin/runner.go

```diff
@@ -13,37 +13,32 @@ func main() {
 
 	engineDetails := utils.EngineDetails{}
 	clients := utils.ClientSets{}
-
 	// Getting kubeConfig and Generate ClientSets
 	if err := clients.GenerateClientSetFromKubeConfig(); err != nil {
 		klog.V(0).Infof("Unable to create ClientSets, error: %v", err)
 		return
 	}
-
 	// Fetching all the ENV's needed
 	utils.GetOsEnv(&engineDetails)
 	klog.V(0).Infoln("Experiments List: ", engineDetails.Experiments, " ", "Engine Name: ", engineDetails.Name, " ", "appLabels : ", engineDetails.AppLabel, " ", "appNamespace: ", engineDetails.AppNamespace, " ", "appKind: ", engineDetails.AppKind, " ", "Service Account Name: ", engineDetails.SvcAccount)
 
+	recorder, err := utils.NewEventRecorder(clients, engineDetails)
+	if err != nil {
+		klog.V(0).Infof("Unable to initiate EventRecorder for Chaos-Runner, would bot be able to add events")
```

Comment:
klog.V(0).Infof("Unable to initiate EventRecorder for Chaos-Runner, would not be able to add events")